### PR TITLE
[MapKit] Update the framework to Xcode 12 beta 4.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/mapkit.cs [hunk 1]
```diff
@@ -1687,6 +1729,28 @@ partial interface MKPolylineRenderer {
...
+		[Export ("strokeEnd")]
+		nfloat StrokeEnd { get; set; }
+	}
+
+	[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+	[BaseType (typeof (MKPolylineRenderer))]
+	partial interface MKGradientPolylineRenderer
+	{
+		[Export ("locations", ArgumentSemantic.Copy)]
+		NSNumber[] Locations { get; }
```
Comment (dalexsoto) [inline src/mapkit.cs:1747]: BindAs?

File: src/mapkit.cs [hunk 2]
```diff
@@ -1687,6 +1729,28 @@ partial interface MKPolylineRenderer {
...
+
+		[Export ("colors", ArgumentSemantic.Copy)]
+		UIColor[] Colors { get; }
+
+		[Export ("setColors:atLocations:")]
+		void SetColors (UIColor[] colors, NSNumber[] locations);
```
Comment (dalexsoto) [inline src/mapkit.cs:1753]: BindAs?

File: src/mapkit.cs [hunk 3]
```diff
@@ -1124,6 +1136,15 @@ interface MKMultiPoint : MKGeoJsonObject {
 
 		[Export ("getCoordinates:range:"), Internal]
 		void GetCoords (IntPtr dest, NSRange range);
+
+		[TV (14,0), NoWatch, Mac (11,0), iOS (14,0)]
+		[Export ("locationAtPointIndex:")]
+		nfloat GetLocation (nuint index);
```
Comment (rolfbjarne) [inline src/mapkit.cs:1142]:
```suggestion
		nfloat GetLocation (nuint pointIndex);
```

File: src/mapkit.cs [hunk 4]
```diff
@@ -2089,4 +2153,75 @@ interface MKMultiPolylineRenderer
 		[Export ("multiPolyline")]
 		MKMultiPolyline MultiPolyline { get; }
 	}
+
+	[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+	[BaseType (typeof (MKAnnotationView))]
+	interface MKUserLocationView { 
+		[DesignatedInitializer]
+		[Export ("initWithAnnotation:reuseIdentifier:")]
+		[PostGet ("Annotation")]
```
Comment (rolfbjarne) [inline src/mapkit.cs:2162]: Why is the PostGet needed?

File: src/mapkit.cs [hunk 5]
```diff
@@ -2089,4 +2153,75 @@ interface MKMultiPolylineRenderer
...
+	[DisableDefaultCtor]
+	interface MKLocalPointsOfInterestRequest : NSCopying
+	{
+		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+		[Field ("MKPointsOfInterestRequestMaxRadius")]
+		double RequestMaxRadius { get; }
+
+		[Export ("initWithCenterCoordinate:radius:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (CLLocationCoordinate2D coordinate, double radius);
```
Comment (rolfbjarne) [inline src/mapkit.cs:2180]:
```suggestion
		IntPtr Constructor (CLLocationCoordinate2D centerCoordinate, double radius);
```

File: src/mapkit.cs (hunk 4 above)
Comment (dalexsoto) [reply src/mapkit.cs:2162]: heh I missed this

**Pull request merged**